src/Main.java

```diff
@@ -24,36 +21,39 @@ public static void main(String[] args) {
                     item.setPrice(scanner.nextDouble());
                     System.out.println("What is the quantity of this item?");
                     item.setQuantity(scanner.nextInt());
+                    id++;
```

Comment:
As discussed this id won't work the way you think.  If you persist the data then you will have duplicate ids because the id would reset every time the program is stopped and started.